test/testrunner.py

```diff
@@ -96,6 +96,8 @@ def main():
     LOGGER.info(f"Images tested: {dlc_images}")
     all_image_list = dlc_images.split(" ")
     standard_images_list = [image_uri for image_uri in all_image_list if "example" not in image_uri]
+    # Do not create EKS cluster for when EIA Only Images are present
+    is_all_standard_images_list_eia = all("eia" in image_uri for image_uri in all_image_list)
```

Comment:
Smart - This accounts for the case where a user might unknowingly trigger EKS tests with EIA images, but it also allows users to run CPU/GPU tests if there are a mix of image types.

`is_all_standard_images_list_eia` is grammatically incorrect.

Also, the name says "standard_images" but the for-loop runs on `all_image_list`.